test/trento/application/integration/discovery/discovery_test.exs

```diff
@@ -83,28 +83,26 @@ defmodule Trento.Integration.DiscoveryTest do
   end
 
   test "should delete events older than the specified days" do
-    for _ <- 0..9 do
-      insert(
-        :discovery_event,
-        agent_id: Faker.UUID.v4(),
-        discovery_type: "discovery_type",
-        payload: %{},
-        inserted_at: Timex.shift(DateTime.utc_now(), days: -11)
-      )
-    end
+    insert_list(
+      10,
+      :discovery_event,
+      agent_id: Faker.UUID.v4(),
```

Comment:
We can safely remove this `agent_id` and `payload` entries. It is generated by the factory